scripts/generate_release_notes.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script copies release notes for the current version from CHANGELOG.md file
+# and stores them in /tmp/release-notes.txt.
+# It also stores them in a /tmp/CHANGELOG_SECTION.md which is used later to copy them over to the
+# CHANGELOG in main branch which is done by merge_changelog_to_main script.
```

Comment:
Nit: Can we include that this script is only being used for package-release.yml workflow?

Reply:
good catch,  I was going to reuse this script in general release script too - updated the PR and tested it in the fork.